### ISSUE TITLE
"A review has been left on your transaction" email sent to reviewer

### ISSUE THREAD

**Issue by lucadgtrip** [OriginProtocol/origin#4359]
I left a review and then got this message

![Screenshot 2020-02-20 at 16 23 50](https://user-images.githubusercontent.com/34341206/74943856-c0851580-53fd-11ea-8012-c0a086136335.png)

which seems strange as it's telling me something I know.

**Comment by franckc** [comment]
The logic deciding what emails to send is [here](https://github.com/OriginProtocol/origin/blob/master/infra/notifications/templates/messageTemplates.js).

We should review it to make sure it makes sense.

**Comment by rembrandtreyes** [comment]
@franckc Yeah looks like we have it reversed. We should flip this to the seller side. Or for two-way visibility, we change the language of this email to be something like, "Your review has been posted to the seller's listing."

I'll review the other email flows to make sure they all make sense.

**Comment by fedealconada** [comment]
I'll grab this one if possible @rembrandtreyes
